pkg/core/helper_test.go

```diff
@@ -52,27 +52,10 @@ var neoOwner = testchain.MultisigScriptHash()
 // examplesPrefix is a prefix of the example smart-contracts.
 const examplesPrefix = "../../examples/"
 
-// newTestChainWithNS should be called before newBlock invocation to properly setup
-// global state.
-func newTestChainWithNS(t *testing.T) (*Blockchain, util.Uint160) {
-	bc := newTestChainWithCustomCfg(t, nil)
-	acc := newAccountWithGAS(t, bc)
-	// Push NameService contract into the chain.
-	nsPath := examplesPrefix + "nft-nd-nns/"
-	nsConfigPath := nsPath + "nns.yml"
-	txDeploy4, _ := newDeployTx(t, bc, acc.PrivateKey().GetScriptHash(), nsPath, nsPath, &nsConfigPath)
-	txDeploy4.Nonce = 123
-	txDeploy4.ValidUntilBlock = bc.BlockHeight() + 1
-	require.NoError(t, addNetworkFee(bc, txDeploy4, acc))
-	require.NoError(t, acc.SignTx(testchain.Network(), txDeploy4))
-	b := bc.newBlock(txDeploy4)
-	require.NoError(t, bc.AddBlock(b))
-	checkTxHalt(t, bc, txDeploy4.Hash())
-
-	h, err := bc.GetContractScriptHash(1)
-	require.NoError(t, err)
-	return bc, h
-}
+const (
+	defaultNameServiceDomainPrice = 10_0000_0000
+	defaultNameServiceSysfee      = 6000_0000
```

Comment:
Maybe we can drop them from `core`? Just add enough GAS to the respective tx for the basic chain.

Reply:
We can, after we make core tests to use the framework.

Reply:
Why can't we do that now? These are contract-specific values, we either should get them at runtime or just add enough GAS until it works. They will get out of sync with the contract itself inevitably and it's not like we care a lot about their precise values.

Reply:
Fixed.